### PR TITLE
Add dynamically resizing of table

### PR DESCRIPTION
**Pull request by cidlik**
This change may be useful for user with non-constant terminal size like me.

### CODE REVIEW

File: logsearch/handlers.py [hunk 1]
```diff
@@ -34,6 +35,7 @@ def __str__(self):
             ]
         )
         t.align = "l"
+        t.max_table_width = os.get_terminal_size().columns - 1
```
Comment (gibizer) [inline logsearch/handlers.py]: As the [doc](https://docs.python.org/3/library/os.html#os.get_terminal_size) suggest we should use [shutil.get_terminal_size](https://docs.python.org/3/library/shutil.html#shutil.get_terminal_size) instead as that handles couple of edge cases better.

**Review by gibizer — CHANGES_REQUESTED** [review]
Thanks you for the contribution. I have one request inline.

File: logsearch/handlers.py (hunk 1 above)
Comment (cidlik) [reply logsearch/handlers.py]: Done.

**Comment by gibizer** [comment]
I'm not sure why it is get autoclosed. I tried to push a rebase but probably mixed it up.

**Comment by gibizer** [comment]
Reopened it as https://github.com/gibizer/zuul-log-search/pull/4

**Pull request closed without merging**